AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3299,8 +3300,8 @@ m.lavenir.net#$#main[role="main"] { transform: translateY(0px)!important; }
 !
 ! Extended CSS rules
 !
-jimmygreen.com##.gdprModal
 myunidays.com#?#.js-toasts > .toast:has(> .message > .message-text > a[data-track="join-cookie-account"])
+galaxus.de#?#body > div#__next > div.ZZ11 > div.ZZZw > div.Z108:has( > .Z10a > .Z10b > p.Z10d:contains(Cookies))
```

Comment:
It's better to change the rule to something like this `galaxus.de#%#document.cookie = "cookieMessageAccepted=1";`